### DISCUSSION TITLE
Should header and footer be visible during printing?

### DISCUSSION THREAD

**Discussion by MikiVanousek** [jpanther/congo#152] [Ideas]
If I print my article (convert it to PDF), I want certain parts of the site to be hidden. I want to use the result as if it were produced using an officer word processing program. Is this just my personal preference, or is this more general? Should I create a pull request hiding those in print? Perhaps as a configurable feature? I can't think of a way to do it on command when printing.

**Comment by jpanther** [comment]
Thanks for the suggestion. This is actually something that I have wanted to add but haven't got around to. If you want to have a go at a PR for this feature I'll happily take a look. Otherwise I can look to make some changes in a future release. I think ideally navigation elements and things should be hidden on print and just show the article content.

Tailwind has a [print modifier](https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles) that could be used to make this quite straightforward.

**Reply by MikiVanousek** [reply]
So you don't think adding a configuration option is necessary / desirable?

**Reply by MikiVanousek** [reply]
> Tailwind has a [print modifier](https://tailwindcss.com/docs/hover-focus-and-other-states#print-styles) that could be used to make this quite straightforward.

The print modifier (and other state modifiers) for some reason do not work for me, even though they appear in the source HTML after I added them to tags in a partial template. I filed an issue on Tailwind, so that their community can help me figure this out. I will submit this PR.

**Reply by MikiVanousek** [reply]
I opened #155.
